media_driver/linux/common/os/i915/mos_bufmgr.c

```diff
@@ -4181,6 +4194,56 @@ int mos_get_slice_mask(uint32_t slice_count)
     return bitmask;
 }
 
+#if defined(MEDIA_EXT)
```

Comment:
This part don't relate to Android at all.

Reply:
I feel so sorry to push wrong code to the repo. I think this time should be right. Thanks you so much